Helper.py

```diff
@@ -0,0 +1,146 @@
+import ROOT
+
+PI=3.14159
+def PassTrig(event,year=2018):
+    
+    TriggerList = []
+    PassTrig = False
+    if (year == 2018):

```

Comment:
@YujiLee301 : For completeness, add all three-year trigger list in *one separate file.* Then import it here.

This separate file should be a [YAML file](https://www.redhat.com/en/topics/automation/what-is-yaml#). One file for each year. Based on the year it will call the appropriate YAML file. And this file supposes to contain all input information that the code will need—for example, Trigger list, Rochester correction file, hard-coded values, etc.